src/Services/Groups.ts

```diff
@@ -0,0 +1,41 @@
+import { AxiosResponse } from 'axios';
+import Service from  './Service';
+import JsonResponse from '../Interfaces/JsonResponse';
+import { Collection, Group } from '../Models/Models';
+import Xuid, { SupportedXuid } from '../Utils/Xuid';
+
+/**
+ * Groups API
+ */
+class Groups extends Service
+{
+    /**
+     * List all groups
+     * 
+     * @returns Group[]
+     */
+    list() {
+        return this.client
+            .get<JsonResponse<Collection<Group>>>(`1/groups`)
+            .then((response: AxiosResponse<JsonResponse<Collection<Group>>>) => {
```

Comment:
Groups endpoints on IAM are now paginated. This is something that has happened since the video so here is the update.

```suggestion
            .then((response: AxiosResponse<PaginatedJsonResponse<Group>>) => {
```